src/styles/index.scss

```diff
@@ -1,3 +1,72 @@
-body {
+@import 'variables';
+@import 'mixin';
+
+@keyframes move {
+  from {
+    transform: rotate(0deg);
+  }
+
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+.page {
   margin: 0;
+  height: 100%;
+
+  @include center_the_element;
+}
+
+.stopwatch {
+  width: $stopwatch_size;
+  height: $stopwatch_size;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+
+  @include center_the_element;
+}
+
+.stopwatch_center {
+  z-index: 1;
+  width: $center_size;
+  height: $center_size;
```

Comment:
Similarly, check if `$center_size` follows the BEM-like naming convention for variables and accurately represents its purpose.